src/components/search-form/search-filters.js

```diff
@@ -9,14 +9,36 @@ import {
 
 import styles from './search-form.css';
 
-export default function SearchFilters({
+const propTypes = {
+  activeFilters: PropTypes.object,
```

Comment:
Could you describe this object?

Reply:
This object is dynamic - keys are filter names and values are filter values. I've left a comment describing the object